site/schemas/create-consequence-operator.schema.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from "zod";
+import { Severity } from "../constants";
+import { setZodDefaultError, zodTimeInHoursAndMinutes } from "../utils";
+
+export const createConsequenceOperatorSchema = z.object({
+    consequenceOperator: z.union(
+        [z.literal("FMAN"), z.literal("SCMN"), z.literal("FSYO"), z.literal("SYRK")],
+        setZodDefaultError("Select at least one operator"),
+    ),
+    description: z.string(setZodDefaultError("Enter a consequence description")).min(1).max(500, {
+        message: "Description must not exceed 500 characters",
+    }),
+    removeFromJourneyPlanners: z.union(
+        [z.literal("yes"), z.literal("no")],
+        setZodDefaultError("Enter planned or unplanned"),
```

Comment:
Would this not be 'Select planned or unplanned' if its a radio button?

Reply:
done